pytorch_lightning/loggers/wandb.py

```diff
@@ -161,11 +164,16 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
         assert rank_zero_only.rank == 0, 'experiment tried to log from global_rank != 0'
 
         metrics = self._add_prefix(metrics)
-        if step is not None and step + self._step_offset < self.experiment.step:
+        if self._sync_step and step is not None and step + self._step_offset < self.experiment.step:
             self.warning_cache.warn(
-                'Trying to log at a previous step. Use `commit=False` when logging metrics manually.'
-            )
-        self.experiment.log(metrics, step=(step + self._step_offset) if step is not None else None)
+                'Trying to log at a previous step. '
+                'Use `WandbLogger(sync_step=False)` or try logging with `commit=False` when calling manually `wandb.log`.')
```

Comment:
```suggestion
                'Trying to log at a previous step.'
                ' Use `WandbLogger(sync_step=False)` or try logging with `commit=False` when calling manually `wandb.log`.')
```
just for consistency with rest of code base